community-tutorials/debootstrap/01-en.md

```diff
@@ -0,0 +1,311 @@
+mount --make-rslave --rbind /sys /mnt/debinst/sys
+mount --make-rslave --rbind /dev /mnt/debinst/dev
+mount --make-rslave --rbind /run /mnt/debinst/run
+
+chroot /mnt/debinst /bin/bash
+```
+
+In the new system we can install everything we need the way we are used to:
+
+```bash
+apt update && apt install -y ssh
+```
+
+## Step 2.1 - (Add backports and security updates to apt)
+
+After the installation is finished we can populate the sources for apt with backports and security updates. First install lsb-release:
+
+```bash
+apt update && apt install -y lsb-release
+```
+
+Apt will occasionally remind you that it won't execute some actions because it knows that we are in a chrooted environment. After that add the sources for your distribution with the following script:
+
+```bash
+CODENAME=$(lsb_release --codename --short)
+cat > /etc/apt/sources.list << HEREDOC
+deb https://deb.debian.org/debian/ $CODENAME main contrib non-free
+deb-src https://deb.debian.org/debian/ $CODENAME main contrib non-free
+
+deb https://security.debian.org/debian-security $CODENAME-security main contrib non-free
+deb-src https://security.debian.org/debian-security $CODENAME-security main contrib non-free
+
+deb https://deb.debian.org/debian/ $CODENAME-updates main contrib non-free
+deb-src https://deb.debian.org/debian/ $CODENAME-updates main contrib non-free
+HEREDOC
+```
+
+It is a good idea to check `/etc/apt/sources.list` manually if everything worked. If everything seems fine then update your system just to be sure:
+
+```bash
+apt update && apt upgrade -y
+```
+
+Continue installing software that you require. For example:
+
+```bash
+apt update && apt install -y zip htop sudo ssh net-tools bash-completion screen iotop rsync 
+```
+
+After you have installed everything you need it is time to add the configuration files. These files can be edited from inside the chrooted system or you can 
+
+```
+exit
+```
+
+to the main system and edit the files from there. I prefer the latter option for two reasons:
+1. It is possible to copy configuration files that already exist.
+2. There is less danger to forget which system you are actually editing since your new system will be at `/mnt/debinst` and you avoid mistakenly editing the wrong file and messing up your backup system.
+
+In this tutorial we will edit the files from the outside. If you do it within a chrooted environment, simply leave out the `/mnt/debinst` at the beginning.
+
+## Step 2.2 - (Edit `/mnt/debinst/etc/fstab`)
+
+The most important configuration is the fstab, to make our new system aware of the partitions it has to mount. While it is possible to call the new installation with `/dev/sdaX`, you should use its UUID as reference in `fstab`. You can find the UUID of a partition via
+```bash
+blkid
+```
+
+As a reference you can copy your previous `fstab` to the new system
+```bash
+cp /etc/fstab /mnt/debinst/etc/fstab
+```
+**Make sure that you at lease change the root partition**
```

Comment:
I think for a better understanding a before and after comparison would be useful.

Reply:
I've added an example fstab. I could add some "magic grep + sed" command but I think this is to dangerous in this case. I've also added a comment to show where the change is necessary